src/utils/quickPickFile.ts

```diff
@@ -178,3 +178,17 @@ export async function quickPickProjectFileItem(context: IActionContext, fileUri:
     }
     return fileItem;
 }
+
+export async function quickPickCsProjFileItem(context: IActionContext, fileUri: vscode.Uri, rootFolder: vscode.WorkspaceFolder, noProjectFileMessage: string): Promise<Item> {
```

Comment:
should we make a new function here that only picks .csproj item? there is an existing method (`quickPickProjectFileItem`) that picks both .csproj and .fsproj right above this method (line 168). We could have an optional parameter in `quickPickProjectFileItem` like `csProjOnly: boolean` to avoid declaring a new method.

i don't feel strongly either way

Reply:
FSProj files might work though we never test with them.

Reply:
thanks for pointing it out. I just checked with Chet, and turned out it does support .fsproj projects. (and also vbproj)

Reply:
We should definitely use the same logic to pick projects that we do for non-SDK builds

Reply:
for non-SDK builds, we can only pick .fsproj and .csproj. but i tested SDK Build with .vbproj and it works too